parsons/civis/civisclient.py

```diff
@@ -21,17 +21,16 @@ class CivisClient(object):
 
     def __init__(self, db=None, api_key=None, **kwargs):
 
-        self.api_key = check_env.check('CIVIS_API_KEY', api_key)
         self.db = check_env.check('CIVIS_DATABASE', db)
+        self.api_key = check_env.check('CIVIS_API_KEY', api_key)
         self.client = civis.APIClient(api_key=api_key, **kwargs)
         """
         The Civis API client. Utilize this attribute to access to lower level and more
         advanced methods which might not be surfaced in Parsons. A list of the methods
         can be found by reading the Civis API client `documentation <https://civis-python.readthedocs.io/en/stable/client.html>`_.
         """  # noqa: E501
 
-    def query(self, sql, preview_rows=10, polling_interval=None, hidden=True,
-              wait=True):
+    def query(self, sql, preview_rows=10, wait=True):
```

Comment:
Good catch on the unused variables. It looks like they are valid arguments to the `query_civis` method on the Civis client (https://civis-python.readthedocs.io/en/stable/generated/civis.io.query_civis.html#civis.io.query_civis). Instead of removing them from the Parsons method, could you update the call to `query_civis` to pass them through?

So, on line 60, something like:
```
fut = civis.io.query_civis(sql, self.db, preview_rows=preview_rows, polling_interval=polling_inverval, hidden=hidden)
```